webapp/channels/build/emoji/make_emojis.mjs

```diff
@@ -443,9 +443,8 @@ const cssRules = `
     background-repeat: no-repeat;
     cursor: pointer;
     -moz-transform: scale(0.5);
+    transform: scale(0.5);
```

Comment:
@lorumic Maybe we can do both here?  Do you mind testing on both?

```suggestion
    zoom: 0.5;  /* fixes blurry images on Chrome? */
    transform: scale(0.5); /* fixes small images on Firefox? */
```

Reply:
The problem is that if you do both, both get applied on FF 133, resulting in the issue that was reported: 0.5 from `zoom` and 0.5 from `transform: scale` results in a final 0.25 size.
I think the takeaway here is that the usage of `zoom` should not be mixed with that of `transform: scale`. You could use `zoom` only, but that would probably affect your browser compatibility: [scale](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scale#browser_compatibility) vs [zoom](https://developer.mozilla.org/en-US/docs/Web/CSS/zoom#browser_compatibility).

Reply:
Ah thank you.  I acknowledge that my knowledge of css is subpar, thus I'm seeking others to help.  I'm just throwing a few ideas.

Perhaps we can use `@supports`?  Again, this may not work as I don't spend much time on web css (I'm more of a mobile engineer here at MM).

```
.zoomable {
  zoom: 0.5; /* Default for browsers supporting zoom */
}

@supports (transform: scale(1)) {
  .zoomable {
    zoom: unset; /* Reset zoom for modern browsers */
    transform: scale(0.5); /* Use scale instead */
    transform-origin: 0 0; /* Anchor scaling */
  }
}
```

Reply:
This is a good idea, but it should be the other way around, as `zoom` is the less-supported property. :)
I have just updated the code to use `zoom` whenever supported (which means "always" in Chrome, where the blurriness problem originally occurred, because Chrome supports it since version 1), and fall back to `transform: scale` when unsupported (e.g. FF versions before 126).
Can you maybe have another look @rahimrahman?

Reply:
Smart, totally see your reasoning of reversing.  Of course, my example would have cause blurriness on Chrome and totally not fixing the problem.  😏 

Do you have a screenshot of Chrome which shows no blurriness and another on Firefox that shows the icon is the correct size?  Visual representation helps a lot, especially for other reviewers.

Thanks for working to solve this even at small **scale** (pun intended!).  I appreciate helping me understand the diff between zoom and scale.  Something I have not encountered before (again as a mobile dev, I don't spend a lot of time here).

Reply:
Just re-read that you're on Linux.  Let me check on mine, or we can get one of our QAs to verify.